www/assessment/stem-assessment.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+//common environment attributes including search paths. not specific to Learnosity
+include_once '../env_config.php';
+
+//site scaffolding
+include_once 'includes/header.php';
+
+//common Learnosity config elements including API version control vars
+include_once '../lrn_config.php';
+
+use LearnositySdk\Request\Init;
+use LearnositySdk\Utils\Uuid;
+
+$security = [
+    'consumer_key' => $consumer_key,
+    'domain'       => $domain
+];
+
+
+//simple api request object for Items API
+$request = [
+    'activity_id' => 'stemactivitiesdemo',
+    'activity_template_id' => 'EMS_DEMO',
+    'name' => 'STEM demo - activities',
+    'rendering_type' => 'assess',
+    'type' => 'local_practice',
```

Comment:
The rest of the demos use `submit_practice` I'd change that for consistency but if you'd prefer to run it as `local_practice` for any reason it's no problem.